amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
```

Comment:
I am afraid that DbSchema does not allow tables to be moved to a different schema...